package.json

```diff
@@ -27,6 +27,7 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "safevalues": "0.5.2",
```

Comment:
not sure what our policy is on using external libraries... @oyj9109 @ChrisAntaki do you know?